src/validator_main.cpp

```diff
@@ -160,15 +160,20 @@ int main(int argc, char** argv)
 
     po::options_description desc = build_command_line_options();
     po::variables_map vm;
-    po::store(po::parse_command_line(argc, argv, desc), vm);
-    po::notify(vm);
+    po::parsed_options parsed = po::command_line_parser(argc, argv).options(desc).allow_unregistered().run();
+    po::store(parsed,vm);
+    std::vector<std::string> unused_args = collect_unrecognized(parsed.options, po::include_positional);
+
+    for (auto a: unused_args) {
```

Comment:
use descriptive names

Reply:
what about unused_parameters or unredognised_options ?

Reply:
oh, sorry, I was referring to the `a` variable, although `unrecognised_parameters` is also better that `unused_args`